client/components/blog/blog-sidebar/blog-sidebar.template.html

```diff
@@ -13,15 +13,14 @@
     </div>
   </div>
 </div>
-/+++
 <div class="blog-category-list">
   <div class="row">
     <div class="col-md-12">
       <h3>Categories</h3>
       <ul>
         <li ng-repeat="category in $ctrl.categoryList  ">
           <a ng-href="#">
-            <i ng-class="{{'fa ' + category.icon}}" aria-hidden="true"></i>
+            <i class="{{'fa ' + category.icon}}" aria-hidden="true"></i>
```

Comment:
As @python4g said, you can create a function that will add `fa ` to your dynamic class and then just use
`ng-class="addFaClass(category.icon)"`
Or try to find different solution.